flank-scripts/src/main/kotlin/flank/scripts/shell/ops/iOSOpsCommand.kt

```diff
@@ -0,0 +1,15 @@
+package flank.scripts.shell.ops
+
+import com.github.ajalt.clikt.core.CliktCommand
+
+object IosOpsCommand : CliktCommand(
+    name = "ios",
+    help = "Build ios test artifacts"
+) {
+
+    override fun run() = listOf(
+            BuildEarlGreyExampleCommand,
+            BuildTestPlansExample,
+            BuildFlankExampleCommand
+        ).forEach(CliktCommand::run)
```

Comment:
would we like to always run all commands?

Reply:
yes